tests/fixtures/fixtures_auth.py

```diff
@@ -42,6 +44,27 @@ def auth_mock(requests_mock):
     return auth
 
 
+@pytest.fixture()
+def auth_account_mock(requests_mock):
```

Comment:
We're settings credit balance and blocks above in project case, why don't we need them here?

Reply:
Because account based authentication is not used for the other tests. If we add to them, we would need to add those mock responses.

Reply:
Actually this was not required by the new testing approach of using both auth_mocks.